crt_portal/cts_forms/templates/forms/complaint_view/disposition/actions/batch/batch_actions.html

```diff
@@ -71,10 +71,18 @@ <h3>Disposal Approval</h3>
         {% endif %}
         <div class="margin-bottom-2 crt-textarea padding-left-2">
             <label for="id_notes">
-              {{ form.notes.label }}
+            {% if not batch.first_reviewer %} {{ form.notes.label }} {% else %} Primary Authorizing Individual's Notes {% endif %}
             </label>
             {{ form.notes }}
         </div>
+        {% if batch.first_reviewer %}
+            <div class="margin-bottom-2 crt-textarea padding-left-2">
+                <label for="id_second_review_notes">
+                {{ form.second_review_notes.label }}
+                </label>
+                {{ form.second_review_notes }}
```

Comment:
What happens if they don't provide notes? Should we should N/A?

Reply:
It would just be an empty field